pkg/block/s3/adapter.go

```diff
@@ -422,6 +431,42 @@ func (a *Adapter) GetPreSignedURL(ctx context.Context, obj block.ObjectPointer,
 	return req.URL, expiry, nil
 }
 
+func (a *Adapter) GetPresignUploadPartURL(ctx context.Context, obj block.ObjectPointer, uploadID string, partNumber int) (string, error) {
+	if a.disablePreSigned {
```

Comment:
```suggestion
	if a.disablePreSigned || a.disablePreSignedMultipart {
```

Reply:
was thinking about this one - if I like to block the presign because of the multipart capability or not. but I kept it bound to the feature it is based on. so if we support presign you can have part presign, in case you need it for some other feature.